tooling/nargo/src/artifacts/debug.rs

```diff
@@ -96,6 +96,32 @@ impl DebugArtifact {
     }
 }
 
+impl From<CompiledProgram> for DebugArtifact {
+    fn from(compiled_program: CompiledProgram) -> Self {
+        DebugArtifact {
+            debug_symbols: vec![compiled_program.debug],
+            file_map: compiled_program.file_map,
+            warnings: compiled_program.warnings,
+        }
+    }
+}
+
+impl From<&CompiledContract> for DebugArtifact {
+    fn from(compiled_artifact: &CompiledContract) -> Self {
+        let all_functions_debug: Vec<DebugInfo> = compiled_artifact
+            .functions
+            .iter()
+            .map(|contract_function| contract_function.debug.clone())
+            .collect();
+
+        DebugArtifact {
+            debug_symbols: all_functions_debug,
+            file_map: compiled_artifact.file_map.clone(),
+            warnings: Vec::new(),
```

Comment:
You should include the warnings from the contract as well

Reply:
ddcf953: chore: include missed warnings